tests/unit/sanakoyeu_et_al_2018/test_data.py

```diff
@@ -0,0 +1,118 @@
+import pytorch_testing_utils as ptu
+import torch
+from torch.utils.data import DataLoader
+
+import pystiche.image.transforms.functional as F
+import pystiche_papers.sanakoyeu_et_al_2018 as paper
+from pystiche.data import ImageFolderDataset
+from pystiche.image import transforms
+from pystiche_papers import utils
+from pystiche_papers.data.utils import FiniteCycleBatchSampler
+from pystiche_papers.utils import make_reproducible
+
+
+def test_image_transform():
+    edge_size = 16
+    make_reproducible()
+    image = torch.rand(1, 1, 800, 800)
+
+    utils.make_reproducible()
+    image_transform = paper.image_transform(edge_size=edge_size)
+    actual = image_transform(image)
+
+    transform = transforms.ValidRandomCrop(edge_size)
+    utils.make_reproducible()
+    desired = F.grayscale_to_fakegrayscale(transform(image))
+
+    ptu.assert_allclose(actual, desired)
+
+
+def test_image_transform_optional_rescale(subtests):
+
+    with subtests.test("too large image"):
```

Comment:
Since no data is shared between three subtests, there is no need for subtests here. Please split this into three separate tests.